rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -117,6 +136,16 @@ def to_exported_dict(
                 exported_dict['asset'] = ''
             if tx_hash is not None:
                 exported_dict['notes'] = f'{evm_explorer}{tx_hash}  ->  {self.notes}'
+            if self.location in EVM_LOCATIONS and database is not None:
+                # call _maybe_add_label_with_address on each address in the note
+                exported_dict['notes'] = re.sub(
+                    pattern=EVM_ADDRESS_REGEX,
```

Comment:
Instead of using this kind of regex matching, since the regex is constant compile the expression and use it: https://docs.python.org/3/library/re.html#re.compile. Since you use it only here you can probably compile it only in this module or well .. wherever you want.